_includes/selector.html

```diff
@@ -480,7 +479,7 @@
                     conda_create_ln,
                     pkgs_ln,
                     add_pkgs_ln
-                ].filter(Boolean).join(" \\\n");
+                ].filter(Boolean).join(" \n");
```

Comment:
```suggestion
                ].filter(Boolean).join(" \\\n");
```

This change should be reverted.

The backslash discrepancy that we were discussing on Slack was on the main [rapids.ai](https://rapids.ai/) site (step 2 under `Quick Local Install`).